### ISSUE TITLE
WARN cannot read initial cursor location

### ISSUE THREAD

**Issue by kfrp** [edgedb/edgedb-cli#101]
- EdgeDB Version: 1.0-alpha.4+g6df187d36.d20200716
- EdgeDB CLI Version: ?
- OS Version: Ubuntu 20.04

Steps to Reproduce:

1. Just install according to instructions at edgedb.com/download

Reporting 2 issues:

1. Every line I type into CLI gives the result, followed by the following block of text:
```
[Timestamp] WARN  rustyline::tty::unix] cannot read initial cursor location
```

2. `\exit` and `\q` don't do anything even though they are recognized as commands.

**Comment by tailhook** [comment]
The (2) is just fixed in e5519c2

The (1) I can't easily reproduce. I've seen it at some point, but don't remember under which conditions. It may happen if something slows up your machine (rustyline waits for the cursor location for just 100ms). Or if you use some weird tool emulating pseudo-tty.

Things to check: Do you use `screen` or `tmux` for this session? Or maybe `script`? Is it over SSH? Is your CPU too loaded? Is it on CPU-constrained VM? 

Generally the warning is harmless and it's safe to ignore.

**Comment by 1st1** [comment]
> Generally the warning is harmless and it's safe to ignore.

Maybe in this case we better hide it in non-debug builds?

**Comment by cosmophile** [comment]
FWIW no RAM constraints or overloading, and no screen or tmux but it is a remote server over SSH from Windows VSCode.

**Comment by tailhook** [comment]
Note to myself: (1) is kinda [fixed in rustyline](kkawakam/rustyline#448), or rather made optional and disabled by default. So unless we find a reason to turn the option on, this will be fixed next time we update rustyline (probably closer to the release, or when indentation issue is settled kkawakam/rustyline#452 / kkawakam/rustyline#453)
